beam-core/beam-core.cabal

```diff
@@ -63,7 +63,7 @@ library
                        time         >=1.6     && <1.10,
                        hashable     >=1.1     && <1.3,
                        network-uri  >=2.6     && <2.7,
-                       containers   >=0.5     && <0.6,
+                       containers   >=0.5     && <0.7,
                        vector-sized >=0.5     && <1.1,
```

Comment:
with the nightly-2018-12-12, I also needed to bump `vector-sized >=0.5     && <1.2`